nullaway/src/main/java/com/uber/nullaway/handlers/OptionalEmptinessHandler.java

```diff
@@ -230,78 +246,97 @@ private boolean optionalIsGetCall(Symbol.MethodSymbol symbol, Types types) {
    * A {@link VariableElement} for a dummy "field" holding the contents of an Optional object, used
    * in dataflow analysis to track whether the Optional content is present.
    */
-  private static VariableElement getOptionalContentElement() {
-    return new VariableElement() {
-      @Override
-      @Nullable
-      public Object getConstantValue() {
-        return null;
+  private static final class OptionalContentVariableElement implements VariableElement {
```

Comment:
Can we add a bit of Javadoc on this class?

Reply:
Sure thing, I kept the javadoc from the `getOptionalContentElement()` factory function which applies to the type, but I should describe that these should be requested from a context rather than instantiated directly.

Reply:
Ah I missed that the previous Javadoc was still there, sorry